reason-react-native/src/components/ComponentMethods.re

```diff
@@ -0,0 +1,45 @@
+type nodeHandle;
+
+type measureError;
+
+type measureResult = {
+  .
+  "x": float,
+  "y": float,
+  "width": float,
+  "height": float,
+  "pageX": float,
+  "pageY": float,
+};
+
+type measureInWindowResult = {
+  .
+  "x": float,
+  "y": float,
+  "width": float,
+  "height": float,
+};
+
+module Make = (T: {type t;}) => {
+  [@bs.module "react-native"] external findNodeHandle: T.t => nodeHandle = "";
```

Comment:
Not too happy about this one. Here I feel that the earlier approach would be nicer where we have a common `Component.ref` to which one can convert using `asComponent`, and `findNodeHandle` just takes a `Component.ref`.